launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0"?>
 <launch>
   <arg name="mode" default="lidar" description="options: `camera_lidar_radar_fusion`, `camera_lidar_fusion`, `lidar_radar_fusion`, `lidar` or `radar`"/>
-  <arg name="publish_rate" default="10.0"/>
   <arg name="enable_delay_compensation" default="true"/>
```

Comment:
[Q]
Could you also remove this default value because it is confusing?

Reply:
This value is also used in radar tracker and I want to share this settings between radar and normal tracker in xml. (This settings can overwrite yaml parameter settings so it may be confusing...